technical-reports/format/groups.md

```diff
@@ -79,7 +79,8 @@ Group keys without a dollar sign (`$`) prefix denote:
     "Group of tokens": {
       "$description": "This is an example of a group containing a single token",
       "Token name": {
-        "$value": "#000000"
+        "$value": "#000000",
```

Comment:
- [ ] Here and blow, for `$type: "color"` tokens, update examples to match current updated format